app/models/Campaign.js

```diff
@@ -34,8 +34,12 @@ const campaignSchema = new mongoose.Schema({
   msg_menu_signedup_external: String,
   msg_menu_signedup_gambit: String,
   msg_no_photo_sent: String,
-  msg_relative_reminder_signup: String,
-  msg_relative_reminder_reportback: String,
+
+  // Exposed messages.
+  messages: {
```

Comment:
Its a message reminding you to do something, relative to another event -- Im not sure of a better naming system?